main.go

```diff
@@ -90,6 +90,7 @@ Options:
   --opt-in-analysis-data                     Send anonymous data for analysis.
   --analysis-data-interval <duration>        Analysis data send interval.
                                               [default: 5m]
+  --packets-v2                               Enable v2 packets (without ids).
```

Comment:
The user won't understand the part about the ids so I think we should remove it. Also add that it's deprecated and that it's left there for compatibility issues